docs/setup.md

```diff
@@ -0,0 +1,120 @@
+## How to setup
+
+This section guides first-time contributor through installing the EvalAI development environment on Ubuntu
```

Comment:
It would be great if we can mention the 'recommended' version of Ubuntu for the contributors.